### PR TITLE
docs: update link to HTMLHint

### PR DESCRIPTION
**Pull request by thedaviddias**
The HTMLHint repository has moved owners and the correct link needs to be updated to reflect this.

### CODE REVIEW

**Comment by Arcanemagus** [comment]
Do you want to rebase this on master? Note that you'll need to run `npm i` again as the dependencies have changed, including new requirements for commit messages.

They now need to follow [Conventional Commits](https://conventionalcommits.org/) in order to support automated releases of new versions. In this case I would make the commit message something like:
```
docs: update link to HTMLHint

The HTMLHint repository has moved owners and the correct link needs to be updated to reflect this.
```

Part of the new dependencies adds a `git` hook that automatically lints your commit message to ensure it follows the required convention 😉. See https://github.com/AtomLinter/linter-htmlhint/pull/178#issuecomment-386783791 for some more information on this, you can change the style if you desire, just as long as it's one that is recognized.

**Comment by thedaviddias** [comment]
Using conventional commits it's something I was planning to deploy on HTMLHint too, so it's perfect!

**Comment by Arcanemagus** [comment]
Reminder that _all_ the commit messages here need to follow the guidelines, so you'll need to rebase the first commit and reword it to follow that 😉.
